draft-sharabayko-srt.md

```diff
@@ -41,9 +46,159 @@ TODO Introduction
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
-document are to be interpreted as described in BCP 14 {{RFC2119}} {{!RFC8174}}
+document are to be interpreted as described in BCP 14 {{RFC2119}} {{RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
+# Data Packets
+
+The data packet structure is as following.
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|F|                    Packet Sequence Number                   |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+| PP|O|Enc|R|                   Message Number                  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                           Timestamp                           |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                     Destination Socket ID                     |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                              Data                             +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+~~~
+{: #datapacket title="data packet structure"}
+
+F (1 bit): 
+: Packet Flag. The flag value of data packet must be 0.
+
+Packet Sequence Number (31 bits):
+: 
+
+PP (2 bits):
+: Position of Packet. This field indicates the position of packet.
+  The value "10b" means the first packet, "00b" is packets in the middle, "01b" is the last packet.
+  If it is a single data packet, the value is "11b".
+
+O (1 bit):
+: Order.  Indicates whether the message should be delivered in order or not.
+  This value is not used in Live Transmission Mode({{live-transmission-mode}}).
+
+Enc (2 bits):
```

Comment:
It was described as `KK`. What does it stand for?

Reply:
KK - Encryption key indicator.